tailwind.config.js

```diff
@@ -29,14 +30,22 @@ module.exports = {
       colors: {
         transparent: `transparent`,
         ds: {
+          background: withOpacity('--color-app-background'),
+          container: withOpacity('--color-app-container'),
+          defaultText: withOpacity('--color-app-text-primary'),
```

Comment:
May be a bit of a big ask, but can we follow tailwind naming convention with kebab case instead of camel case, for these custom class names/ For instance here it would be `'default-text': ...`

Reply:
feel free to ignore the lint rule for this whole file

Reply:
Fixed! https://github.com/codecov/gazebo/pull/3136/commits/4a96955744a3dc740fc35c75dc487c24082a52c5
Seems lint rule is okay with it!

Reply:
Woo 🎉 awesome